### PR TITLE
KOGITO-3684 Handle error response in node trigger

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

Description: This PR contains changes to handle error response during the retrieval of nodes.

JIRA : https://issues.redhat.com/browse/KOGITO-3684

### CODE REVIEW

**Comment by AjayJagan** [comment]
jenkins please retest this

**Comment by AjayJagan** [comment]
jenkins please retest this

**Comment by AjayJagan** [comment]
jenkins please retest this

File: ui-packages/packages/management-console/src/components/Organisms/ProcessDetailsNodeTrigger/ProcessDetailsNodeTrigger.tsx [hunk 1]
```diff
@@ -45,12 +45,25 @@ const ProcessDetailsNodeTrigger: React.FC<IOwnProps & OUIAProps> = ({
   const [modalTitle, setModalTitle] = useState<string>('');
   const [titleType, setTitleType] = useState<string>('');
   const [modalContent, setModalContent] = useState<string>('');
-  const [nodes, setNodes] = useState([]);
+  const [error, isError] = useState<boolean>(false);
```
Comment (jstastny-cz) [inline ui-packages/packages/management-console/src/components/Organisms/ProcessDetailsNodeTrigger/ProcessDetailsNodeTrigger.tsx]: isError should be a setter, right? I would name it setError to be in sync with rest of state props.

Comment (AjayJagan) [reply ui-packages/packages/management-console/src/components/Organisms/ProcessDetailsNodeTrigger/ProcessDetailsNodeTrigger.tsx]: changed the setter name

**Review by nmirasch — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]
LGTM, thanks.

**Review by Sara4994 — APPROVED** [review]

**Comment by Sara4994** [comment]
LGTM :+1:

**Pull request merged**